hive3/build.gradle

```diff
@@ -16,6 +16,9 @@
  * specific language governing permissions and limitations
  * under the License.
  */
+def hiveVersion = "3.1.2"
```

Comment:
Maybe a comment here that we override the original values for hive3?

Reply:
Sure, done